cmd/collector/main.go

```diff
@@ -58,7 +62,10 @@ func main() {
 	if err != nil {
 		log.Fatalf("Cannot initialize storage factory: %v", err)
 	}
-
+	strategyStoreFactory, err := ss.NewFactory(ss.FactoryConfigFromEnv())
```

Comment:
This might be a breaking change. If no Env is provided, we default to using the default probability of 0.001 (all client's remote controlled samplers default to using 0.001 if it can't contact the collector...)

Do we want to have no default and not initialize a handler if Env is not provided and have users manually enable?

Reply:
Sorry, how is it "breaking"? Sounds like you're saying the effect of no Env will be the same - 0.001